### DISCUSSION TITLE
FHIR resource "No Models Found"

### DISCUSSION THREAD

**Discussion by marcheschi** [nextgenhealthcare/connect#4887] [General]
Hi 
I migrated a 3.9.1 mirth installation to a new Vm with different IP, and when I try to create a fhir message builder I got:
![image](https://user-images.githubusercontent.com/3910408/145042381-70fe6819-434e-4fcb-9ac1-3127e4f72bd8.png)

Where is gone the resources? 
Is there a way to re-enable it?
Thank you
Paolo

**Comment by pacmano1** [comment]
What do you mean by "migrated".    A VM snapshot and move?  Also, the FHIR plugin is no longer free - so I assume you have licensed the Gold license pack.

**Reply by marcheschi** [reply]
Yes a vm snapshot and move.
This is only a vm I use to learn FHIR I do not have a license.
So the plugin is not usable anymore? also for research or educational projects?
Thank you
Paolo

**Reply by pacmano1** [reply]
I will defer to Nextgen on their license restrictions, but I suspect that no, you cannot.

**Reply by marcheschi** [reply]
Ok I deleted it.
I will use some other opensource product to learn fhir.
Thank you
Paolo
